src/layouts/partials/menu.html

```diff
@@ -19,7 +19,21 @@
                                         {{ if $currentPage.IsDescendant $pageForItem2 }}
                                             <ul class="acnav__list acnav__list--level3">
                                                 {{ range .Children }}
-                                                <li><a class="acnav__link{{ if $currentPage.IsMenuCurrent "main" . }} current{{ end }}" href="{{ .URL }}">{{ .Name }}</a></li>
+													{{ $pageForItem3 := $site.GetPage .URL }}
```

Comment:
Added this stuff to have nested items in menu, like:
<img width="251" alt="obraz" src="https://user-images.githubusercontent.com/39192420/187739899-13f9d3d6-cdd5-41b2-bdbf-9d86c3dc088d.png">
Without it, the `gitops` section was empty, though links were accessible.